docs/databricks.md

```diff
@@ -0,0 +1,6 @@
+# Running Quilt in Databricks
+
+## Install `quilt3` on Your Cluster
+Before using `quilt3` in your Databricks notebooks, you'll need to install it on your cluster. Find your cluster from the __Compute__ tab. From your Cluster overview, click on the __Libraries__ tab, then click the "Install new" button. Select PyPI under Library Source and enter `quilt3` for the Package.
```

Comment:
Please add a paragraph with more setup on why people would use Quilt with Databricks, how functions complement, Quilt is compute agnostic, etc.